x/lockup/keeper/synthetic_lock_test.go

```diff
@@ -0,0 +1,311 @@
+package keeper_test
+
+import (
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/osmosis-labs/osmosis/x/lockup/types"
+)
+
+func (suite *KeeperTestSuite) TestSyntheticLockupCreateGetDeleteAccumulation() {
```

Comment:
Can we make all the tests in this file have much less duplicate test setup code & preferrably be table driven. 

I don't think we should have this much duplicate test setup code going forward, it becomes too hard to extend and too hard to understand

Reply:
There were lots of test modifications on superfluid minimum viable version and I would like to implement this comment on that PR